PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+# Feature or Problem
+Briefly describe the reason for this pull request: the feature being added or problem being solved.
```

Comment:
Recommendation, change to comments so that we never need to delete these.

<!--- Briefly describe the reason for this pull request: the feature being added or problem being solved.-->